servers/rendering/rendering_device_graph.cpp

```diff
@@ -1517,6 +1626,24 @@ void RenderingDeviceGraph::begin() {
 #endif
 }
 
+void RenderingDeviceGraph::add_acceleration_structure_build(RDD::AccelerationStructureID p_acceleration_structure, ResourceTracker *p_dst_tracker, const Vector<ResourceTracker *> &p_src_trackers) {
```

Comment:
VectorView is preferred for the input here so we don't impose which type of Vector we allow as the input.